src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 100%;
+  max-width: 200px;
+  margin-bottom: 50px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $back-color;
```

Comment:
If you only need a color - use the exact property - `background-color`.
This will make your code more understandable

Reply:
Yes, you are right, thanks.